ogarniacz_mp3.py

```diff
@@ -38,6 +40,21 @@ def get_format_preference_index(ext):
     except ValueError:
         return len(format_preference)
 
+
+def convert_to_mp3(processing_dir_path):
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path):
+        for file_name in files:
+            file_path = os.path.join(dir_path, file_name)
+            if classes.MusicFile.is_music_file(file_path):
+                name, ext = split_file_name(file_name)
+                if ext != "mp3":
+                    input_path = file_path
+                    output_path = os.path.join(dir_path, name + ".mp3")
+                    subprocess.call(["ffmpeg", "-i", input_path, "-vn", "-ar", 44100, "-ac", 2, "-b:a", 142000, output_path],
+                    stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE)
+                    os.remove(file_path)
```

Comment:
Przed remove musisz sprawdzić, czy konwersja zakończyła się powodzeniem.
```
if proc.returncode == 0:
    os.remove(file_path)
else:
    print(stderr.decode())
```

Reply:
Brawo!

Reply:
Do czego ma się odnosić `proc`? PyCharm mówi, że nie jest zdefiniowane.
I o jakie `stderr` chodzi? To też podkreśla i sugeruje zaimportowanie - `from sys import stderr` - tylko wtedy podkreśla `decode()` i daje taki komunikat: `Unresolved attribute reference 'decode' for class 'TextIO' `.

Reply:
Zgubne skutki przerabiania starego kodu.
Widzę, że `subprocess.call` wychodzi z użycia i należy stosować `subprocess.run`.

Więc tak:
```
proc = subprocess.call(["f itp. itd.
# i tutaj można już to co powyżej.
if proc.returncode == 0:
    os.remove(file_path)
else:
    print(proc.stderr.decode())
```